app/models/featured_topic.rb

```diff
@@ -226,9 +226,9 @@ class FeaturedTopic
       title: "Periodic Tables",
       description: "This digital collection features selected visual representations of the periodic table of the elements, with an emphasis on alternative layouts including circular, cylindrical, pyramidal, spiral, and triangular forms. Ranging in date from the 1860s to the 1990s, the materials provide a panorama of the historic evolution of the periodic table following Dmitri Mendeleev's initial 1869 design. Illustrations depicting chemical elements and concepts in atomic theory are also included in this collection."
     },
-    science_on_stamps: {
-      subject: ["Science on postage stamps"],
-      description: "This digital collection features selected postage stamps and other philatelic materials depicting various aspects of science: from molecules and chemical reactions to portraits of Nobel Prize laureates. Materials in this collection, dating from 1887 to 2000, include stamps and first day covers, as well as commemorative and postmarked envelopes and postcards. Most of the items were selected from the Witco Stamp Collection in the Institute's archives."
+    :oral_histories => {
+      title: "Oral Histories",
+      path: "/collections/gt54kn818"
```

Comment:
The trick is that ideally this would use our ENV variable for the collection ID, instead of hard-coding it.  Everywhere else we use that variable...

actually that's no trick at all, it should be easy to do that here, since it's an actual ruby file? I was at first thinking it was a YAML where that would be harder, but it's not!  Unless we run into weird problems with order-of-operations on boot, should work fine, please try to use the ScihistDigicoll::Env lookup for id!

Reply:
Good call. Works great!